source/conf.py

```diff
@@ -49,10 +57,15 @@
     'q2doc.qiime1',
     'q2doc.checkpoint',
     'q2doc.command_block',
-    'q2doc.plugin_directory',
     'q2doc.external_links',
 ]
 
+
+generate_plugin_rst()
+
+root = pathlib.Path(__file__).parent
+bibtex_bibfiles = [str(p.relative_to(root)) for p in root.rglob("*bib")]
```

Comment:
I think maybe we should make it slightly more precise, and include the `.` delimiting the file's name and extension.

```suggestion
bibtex_bibfiles = [str(p.relative_to(root)) for p in root.rglob("*.bib")]
```